generator/api/src/main/java/io/fabric8/maven/generator/api/FromSelector.java

```diff
@@ -43,12 +45,15 @@
 
     private final GeneratorContext context;
 
+    private final Logger logger;
+
     private final Pattern REDHAT_VERSION_PATTERN = Pattern.compile("^.*\\.(redhat|fuse)-.*$");
 
     private Plugin plugin;
     private Plugin compilerPlugin;
-    public FromSelector(GeneratorContext context) {
+    public FromSelector(GeneratorContext context, Logger logger) {
```

Comment:
This is not the way to initialize loggers.

`private static final Logger log = LoggerFactory.getLogger(FromSelector.class);`